rust/datafusion/tests/sql.rs

```diff
@@ -1530,16 +1530,11 @@ async fn execute(ctx: &mut ExecutionContext, sql: &str) -> Vec<Vec<String>> {
 
     let msg = format!("Creating physical plan for '{}': {:?}", sql, plan);
     let plan = ctx.create_physical_plan(&plan).expect(&msg);
-    let physical_schema = plan.schema();
 
     let msg = format!("Executing physical plan for '{}': {:?}", sql, plan);
     let results = collect(plan).await.expect(&msg);
 
     assert_eq!(logical_schema.as_ref(), optimized_logical_schema.as_ref());
-    assert_eq!(
```

Comment:
This is interesting as it assumes none of the attributes can change 'dynamically' at runtime. I.e. if we `TRY_CAST` a column it can always return a `NULL`. This does not support runtime evaluation.